UPGRADE_6.0.md

```diff
@@ -0,0 +1,87 @@
+UPGRADE FROM 5.x to 6.0
+=======================
+
+## Introdcure ES6
```

Comment:
We were not planning to drop the support for IE11 yet. It's sad the we can not use the template literals cause of that, I am also a fan of them. Since I know there is also no polyfill for this.